docs/stubs.rst

```diff
@@ -947,8 +944,8 @@ Maybe::
         def foo(self) -> list[int]: ...
         def bar(self) -> Mapping[str]: ...
 
-Avoid ``Union`` return types, since they require ``isinstance()`` checks.
-Use ``Any`` if necessary.
+Avoid union return types, since they require ``isinstance()`` checks.
+Use ``Any`` or ``X | Any`` if necessary.
```

Comment:
It would be useful to explain more about how the `X | Any` trick works: when should you use it? what does it do?

Reply:
Indeed, but as this is not stubs-specific, probably in a best practices document?